init.el

```diff
@@ -15,6 +15,7 @@
 ;; Avoid garbage collection during startup.
 ;; see `SPC h . dotspacemacs-gc-cons' for more info
 (defconst emacs-start-time (current-time))
+(setq gnutls-algorithm-priority "NORMAL:-VERS-TLS1.3")
```

Comment:
Why are you disabling TLS 1.3? Please remove this line

Reply:
:+1:  
yeah, I was a bit lazy, you often just cherry pick the commits so I didn't bother to remove a local fix for some intermittent issues.. I can remove the commit from the branch